src/main/java/edu/RobertoAranovich/advancedjava/BasicStockService.java

```diff
@@ -0,0 +1,30 @@
+package edu.RobertoAranovich.advancedjava;
+
+import java.math.BigDecimal;
+
+/**
+ * Basic implementation of StockService interface.
+ *
+ * @author Roberto Aranovich
+ */
+
+public class BasicStockService implements StockService {
+    private BigDecimal price = new BigDecimal(171.44);
+
+    /**
+     * Non argument constructor.
+     */
+    public BasicStockService() {
+    }

```

Comment:
FYI - you do not need this constructor here. Can you tell me why?

Reply:
It will use a default constructor from the superclass (Object), I assume...